docs/workflows/authentication.rst

```diff
@@ -0,0 +1,75 @@
+.. _authentication:
+
+Registry Token Authentication
+=============================
+
+Pulp supports pulling content from **authenticated registry**. This enables users to pull content with
```

Comment:
I suggest to change this so it does not sound confusing:
s/Pulp supports pulling content from **authenticated registry**/Pulp Registry supports token auth.